app/policies/planning_application_policy.rb

```diff
@@ -18,4 +18,18 @@ def edit?
   def update?
     super || signed_in_editor?
   end
+
+  def unpermitted_statuses
+    PlanningApplication.statuses.keys - permitted_statuses
+  end
+
+  def permitted_statuses
+    if @user.assessor?
+      [ "awaiting_determination" ]
+    elsif @user.reviewer? || @user.admin?
```

Comment:
Are we sure that the reviewer should not be permitted to do this? We would normally consider a manager to have a super-set of permissions rather than a different set of permissions? (I'm aware that this is more of a product question than a code question).

Reply:
I guess the way I was thinking about this was that we don't currently have a UI for the manager to push something from in assessment to awaiting determination. So I wanted to block things until we know otherwise.

We do have an upcoming story for a manager to put something back into 'in_assessment' (asking the officer to re-evaluate), so we'll definitely need to revisit this then if not before.